src/pages/player/player.jsx

```diff
@@ -0,0 +1,104 @@
+var React = require('react');
+var {parseJSON} = require('../../../node_modules/webmaker-core/src/lib/jsonUtils');
```

Comment:
That sounds like a good idea, but I think we should do it later in a follow up ticket. Want to make one?